test/plugin/test_out_google_cloud.rb

```diff
@@ -1160,80 +895,369 @@ class Fluent::GoogleCloudOutput # rubocop:disable Style/ClassAndModuleChildren
     public :parse_severity
   end
 
-  def test_parse_severity
-    test_obj = Fluent::GoogleCloudOutput.new
+  def test_parse_severity
```

Comment:
Ah.... the diff looks ugly from here. The change is just to move all the helper functions and constants to the "private" section.